docs/source/deployment/airflow.md

```diff
@@ -166,6 +166,69 @@ astro deploy
 
 ![](../meta/images/astronomer_cloud_deployment.png)
 
+## How to run a Kedro pipeline on Amazon AWS Managed Workflows for Apache Airflow (MWAA)
+
+### Kedro project preparation
+MWAA, or Managed Workflows for Apache Airflow, is an AWS service that makes it easier to set up, operate, and scale Apache Airflow in the cloud. Deploying a Kedro pipeline to MWAA is similar to Astronomer, but there are some key differences: you need to store your project data in an AWS S3 bucket and make necessary changes to your Data Catalog. Additionally, you must configure how you upload your Kedro configuration, install your Kedro package, and set up the necessary environment variables.
+1. Complete steps 1-4 from the [Create, prepare and package example Kedro project](#create-prepare-and-package-example-kedro-project) section.
+2. Your project's data should not reside in the working directory of the Airflow container. Instead, [create an S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-bucket.html) and [upload your data folder from the new-kedro-project folder to your S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/upload-objects.html).
+3. Modify the Data Catalog to reference data in your S3 bucket by updating the filepath and add credentials line for each Dataset in the `new-kedro-project/conf/airflow/catalog.yml`. Add the S3 prefix to the filepath as shown below:
+```shell
+companies:
+  type: pandas.CSVDataset
+  filepath: s3://your_S3_bucket/data/01_raw/companies.csv
+  credentials: dev_s3
+```
+4. [Set up AWS credentials](https://docs.aws.amazon.com/keyspaces/latest/devguide/access.credentials.html) to provide read and write access to your S3 bucket. Update `new-kedro-project/conf/local/credentials.yml` with your AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY:
+```shell
+dev_s3:
+  client_kwargs:
+    aws_access_key_id: *********************
+    aws_secret_access_key: ******************************************
+```
+5. Add s3fs to your project’s `requirements.txt` in `new-kedro-project` to facilitate communication with AWS S3.
+```shell
+s3fs
+```
+
+6. Archive your `conf` folder into a `conf.zip` file and upload it to `s3://your_S3_bucket` for later use in the Airflow container. This file will be unzipped into the `plugins` folder within the MWAA Airflow container.
```

Comment:
Worth noting that the `kedro package` command will create an archived `conf-<package-name>.tar.gz` file in the `dist/` folder without the local conf which should be used here ^

Reply:
Do you have any suggestions on how this file should be used? Should we simply unpack it to extract the `conf` folder, or should we run `kedro run` directly with the `conf` archive?

Reply:
It doesn't need to be unpacked at all, `OmegaConfigLoader` can handle `.tar.gz` and `.zip` confs

Reply:
I changed that but I don't know how to be with `export KEDRO_LOGGING_CONFIG="plugins/logging.yml"`, I currently copy `logging.yml` separetely, but maybe you know better solution.